packages/opencensus-core/src/trace/model/no-record/no-record-root-span.ts

```diff
@@ -35,22 +34,25 @@ export class NoRecordRootSpan extends NoRecordSpanBase implements
   /**
    * Constructs a new NoRecordRootSpanImpl instance.
    * @param tracer A tracer object.
-   * @param context A trace options object to build the no-record root span.
+   * @param name The displayed name for the new span.
+   * @param kind The kind of new span.
+   * @param traceId The trace Id.
+   * @param parentSpanId The id of the parent span, or empty if the new span is
+   *     a root span.
+   * @param traceState Optional traceState.
    */
-  constructor(tracer: types.Tracer, context?: types.TraceOptions) {
+  constructor(
+      tracer: types.Tracer, name: string, kind: types.SpanKind, traceId: string,
```

Comment:
Should these new parameters be optional to make this less of a breaking change? If we keep them, could you modify the change list to include a mention of it?

Reply:
Possible. On the other hand, this is internal API and won't affect users in any way.

Reply:
OK, I see, this does not change the `RootSpan` interface type, just the implementations. In OpenCensus Web, I create spans directly from performance timings, but it seems like the instrumented Node libraries use the helper utilities in `Tracer` to create, start and end spans. So makes sense this is not really part of the exported API.